2-mandatory/1-create-functions.js

```diff
@@ -24,16 +27,30 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
+function removeExtraThings(word) {
+  let newWord;
```

Comment:
In this case, you're not re-assigning any new value to newWord. so it's better just declair&assign at the same time 

`const newWord = word.trim().replace('/', '').toLowerCase();`

Reply:
function removeExtraThings(newword) {
  const newWord = newword.trim().replace('/', '').toLowerCase();
  return newWord;
}